pyftpdlib/test/test_functional.py

```diff
@@ -560,6 +560,41 @@ def test_mdtm(self):
         else:
             self.fail('Exception not raised')
 
+    def test_mfmt(self):
+        # making sure MFMT is able to modify the timestamp for the file
+        test_timestamp = "20170921013410"
+        self.client.sendcmd('mfmt ' + test_timestamp + ' ' + self.tempfile)
+        resp = self.client.sendcmd('mdtm ' + self.tempfile)
+        self.assertTrue('213' in resp and test_timestamp in resp)
```

Comment:
Please use `self.assertIn` instead (clearer message in case of error).
Also, in addition to this, use `os.path.getmtime` against the physical file and make sure the time is what you expect.